.github/workflows/merge-queue.yml

```
@@ -47,8 +47,174 @@ jobs:
     needs:
       - pre-checks
+  docker-build:
+    needs:
+      - prepare
+      - pre-checks
+    runs-on: arc-runners
+    permissions:
+      actions: read
+      contents: read
+    outputs:
+      json: ${{ steps.matrix-output.outputs.json }}
```

actionlint (1.7.4)

property "matrix-output" is not defined in object type {cache-check: {conclusion: string; outcome: string; outputs: {string => string}}; dockerargs: {conclusion: string; outcome: string; outputs: {string => string}}; dockerbuild: {conclusion: string; outcome: string; outputs: {string => string}}; gather: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)